Core/src/IfcGeometryConverter/ProfileCache.h

```diff
@@ -95,6 +97,7 @@ namespace OpenInfraPlatform {
 				protected:
 
 					std::shared_ptr<PlacementConverterT<IfcEntityTypesT>> placementConverter;
+					std::shared_ptr<SplineConverterT<IfcEntityTypesT>> splineConverter;

```

Comment:
Similarly, remove the member variable, as it is not needed.

Reply:
Done in commit b13e888